### ISSUE TITLE
I am getting error in fedora

### ISSUE THREAD

**Issue by Ashish882** [Screetsec/TheFatRat#167]
When I install the fatrat in fedora it give following error solved this error plz....

<code>64Bit OS detected

 Checking type of shell ....
[local]
./setup.sh: line 1136: ps: command not found
 </code>
![screenshot_20170808-102238](https://user-images.githubusercontent.com/30336226/29056836-8f606d1a-7c23-11e7-9b27-97ea37baf859.png)

**Comment by peterpt** [comment]
you should not use fatrat in an android device , we have no way to test it properly .
PS is an essential tool that comes in every linux distribution .
Try installing by issuing this command on terminal .
dnf install procps-ng

fatrat is mainly a debian based tool , which use apt-get to install packages , and i have no clew if you will be able to install it properly .

You should read the readme file in fatrat before starting creating issues on linux OS that we did not wrote that it work .

From readme.md 
**A linux operating system. We recommend :**

    Kali Linux 2 or Kali 2016.1 rolling
    Cyborg
    Parrot
    BackTrack
    Backbox
    Devuan

Basically we test it only on debian based linux and arch linux .
